packages/node_modules/@ciscospark/widget-space/src/enhancers/setup.js

```diff
@@ -64,7 +64,7 @@ function setup(props, prevProps) {
     if (!mercuryStatus.hasConnected
       && !mercuryStatus.connecting
```

Comment:
I don't know why, but I hate this syntax

Reply:
Open to suggestions for change. One thing I thought about was to make it a function or variable so it is easier to read like:

```js
const shouldConnectToMercury = !mercuryStatus.hasConnected && !mercuryStatus.connecting && !mercuryStatus.connected && sparkState.registered;

if (shouldConnectToMercury) {
```